docs/src/components/ComponentDoc/ComponentControls/ComponentControlsCodeSandbox/ComponentControlsCodeSandbox.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react'
+import CodeSandboxer from 'react-codesandboxer'
+import { Menu } from 'semantic-ui-react'
+
+import pkg from 'package.json'
+import { ComponentSourceManagerLanguage } from 'docs/src/components/ComponentDoc/ComponentSourceManager'
+import { updateForKeys } from 'docs/src/hoc'
+import { appTemplateJs } from './indexTemplates'
+
+const dependencies = {
+  '@stardust-ui/react': pkg.version,
```

Comment:
unfortunately, it won't work for localhost :(

Reply:
Yep, this is unfixable because CodeSandbox installs dependencies from NPM 🐤